docs/faq.md

```diff
@@ -204,6 +336,99 @@ You can consult booking made through Flight Create Orders using the Flight Order
+
+
+## Hotel Booking
+
+### What type of payments are supported?
+
+The current version of the Hotel Booking API only supports credit card payments. The Hotel Search API returns the payment policy of each hotel under `acceptedPayments` in the policies section.
+
+### Can I markup the room prices?
+
+It is not possible to markup the prices of the hotel rooms with the current version of the Hotel Booking API. The reason is that the content we offer today in our Hotel Search/Book API is post-paid, meaning the traveler will pay directly at the hotel. The Hotel Booking API is here to enable making a reservation but not to pay directly. We are working on adding more hotel offers, especially offers that will be pre-paid, meaning you will be able to charge the travelers directly and add a markup. However, you still need to add a credit card while booking in case of cancellations or no-shows.
+
+### Payment providers and gateways
+
+The Hotel Booking API works by using the guest's payment information and sending it to a chosen hotel for the reservation. You can use a payment gateway in your app, but this will not change the way the API works. The hotels will never collect any money from you. Instead, the payments are always done at the time of checkout between the guest and the hotel. During the booking process, Amadeus passes the payment and guest information to the hotel but does not validate information, so it doesn’t play the role of payment gateway. Be sure to validate the payment and guest information as invalid information may result in the reservation being canceled. As soon as your application stores, transmits, or processes cardholder information, you will need to comply with the PCI Data Security Standard (PCI DSS). For more information, visit the [PCI Security Council website](https://www.pcisecuritystandards.org/merchants/).
+
+### How can I cancel a room booking?
+
+As of now, the hotel booking API does not allow canceling rooms. If this option is possible with your hotel offer, the cancellation will have to be done manually with the hotels. We are working on an API for the cancellation; however, it is still too soon to commit to anything.
+
+### Why do I get 500 status code?
+
+The process of booking a hotel in the test environment involves sending your request to each hotel provider, and each provider has its own environment and rules. Due to these differences, there may be connectivity issues with the providers that can result in a timeout. Additionally, if many requests are sent to a particular hotel, they may choose to block them. If you provide us with a timestamp and details of another API request that has failed, including the hotel in question, we can search our logs to find more information. However, it is likely that the issue is one of the aforementioned cases.
+
+### How can I see Amadeus API coverage for a hotel chain?
+
+You can use the Hotel Search API to search the hotels by their hotel Id.  
```

Comment:
You can find the list of supported hotel chains in our data collection: https://github.com/amadeus4dev/data-collection/blob/master/data/hotelchains.md.